src/lib/api/org/index.ts

```diff
@@ -168,7 +177,7 @@ export async function listProjects(
   try {
     const res = await requestManager.request({
       verb: 'post',
-      url: `/org/${orgId.trim()}/projects`,
+      url: `${SNYK_API}/orgs/${orgId.trim()}/projects?version=2021-06-04`,
```

Comment:
this will not work, the request manager is already calling `v1` here. You need to have a look how to configure the base to the request Manager, right now you are calling a url that looks like this:

https://snyk.io/api/v1/https://api.snyk.io/rest/orgs/${orgId.trim()}/projects?version=2021-06-04

Reply:
This URL will be incorrect too with Rest support https://github.com/snyk-tech-services/snyk-api-import/pull/342/files#diff-64b15bd64ec57f1c1662d5a8d9ed957cea2aef392bc017034bb8f01b204c63f4R172-R174

Reply:
I'll update the version. I ran in debug mode yesterday actually I had those log:  
```
follow-redirects options {
  maxRedirects: 21,
  maxBodyLength: 10485760,
  protocol: 'https:',
  path: '/rest/orgs/74e2f385-a54f-491e-9034-76c53e72927a/projects?version=2021-06-04',
  method: 'POST',
  headers: {
    Accept: 'application/json, text/plain, */*',
    'Content-Type': 'application/json',
    Authorization: 'token 744cabd5-4449-4d9b-bb71-c56cb3ef4ee7',
    'User-Agent': 'scripts/snyk-api-import/tech-services/snyk-request-manager/1.0',
    'Content-Length': 24
  },
  agent: undefined,
  agents: { http: undefined, https: undefined },
  auth: undefined,
  hostname: 'api.snyk.io',
  port: null,
  nativeProtocols: {
    'http:': {
      _connectionListener: [Function: connectionListener],
      METHODS: [Array],
      STATUS_CODES: [Object],
      Agent: [Function],
      ClientRequest: [Function: ClientRequest],
      IncomingMessage: [Function: IncomingMessage],
      OutgoingMessage: [Function: OutgoingMessage],
      Server: [Function: Server],
      ServerResponse: [Function: ServerResponse],
      createServer: [Function: createServer],
      get: [Function],
      request: [Function],
      maxHeaderSize: [Getter],
      globalAgent: [Getter/Setter]
    },
    'https:': {
      Agent: [Function: Agent],
      globalAgent: [BoundHttpsProxyAgent],
      Server: [Function: Server],
      createServer: [Function: createServer],
      get: [Function],
      request: [Function]
    }
  }
} +1
```

Reply:
Will trying something else anyways

Reply:
is apiv3 enabled for the group we are testing?

Reply:
also I tried manually (using the apiv3 doc) yesterday and had the same error